src/applications/discharge-wizard/components/InstructionsPage.jsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
-import { Link } from 'react-router';
 
-const InstructionsPage = () => {
+const InstructionsPage = ({ router }) => {
+  const startForm = event => {
+    event.preventDefault();
+    router.push('questions');
```

Comment:
Should this be using a constant?

Reply:
I think so? Using an arrow function. What else could I use here?

Reply:
I meant the `'questions'` string - should this be referring to a constant somewhere?

Reply:
Oh my mistake. Sorry about that. Let me create them. I need more coffee...